### PR TITLE
Enforce minimum Dokan version

### PR DESCRIPTION
**Pull request by zanderz**

### CODE REVIEW

File: dokan/loaddll.go [hunk 1]
```diff
@@ -32,9 +34,49 @@ func (ep *errorPrinter) Printf(s string, os ...interface{}) {
 	fmt.Fprintf(&ep.buf, s, os...)
 }
 
+// adapted from keybase/client/go/install/fuse_status_windows.go
+func isDokanCurrent(path string) (bool, error) {
```
Comment (taruti) [inline dokan/loaddll.go]: Can we ever return "false, nil" ?

If not then perhaps just ```func isDokanCurrent(path string) error```.

File: dokan/loaddll.go [hunk 2]
```diff
@@ -32,9 +34,49 @@ func (ep *errorPrinter) Printf(s string, os ...interface{}) {
...
 // loadLibrary calls win32 LoadLibrary and logs the result.
 func loadLibrary(epc *errorPrinter, path string) (windows.Handle, error) {
 	hdl, err := windows.LoadLibrary(path)
+	if err == nil {
+		current, err2 := isDokanCurrent(path)
+		if !current {
+			windows.Close(hdl)
+		}
+		err = err2
+	}
```
Comment (taruti) [inline dokan/loaddll.go]:
```
func loadLibrary(epc *errorPrinter, path string) (windows.Handle, error) {
	hdl, err := windows.LoadLibrary(path)
	epc.Printf("LoadLibrary(%q) -> %v,%v\n", path, hdl, err)
	if err!=nil {
		return hdl, err
	}
	err = isDokanCurrent(path)
	if err!=nil {
			windows.Close(hdl)
		}
	}
	return hdl, err
}
```

Comment (zanderz) [reply dokan/loaddll.go]: ok

**Comment by taruti** [comment]
https://github.com/dokan-dev/dokany/issues/765 might come early enough for us.

**Comment by taruti** [comment]
https://github.com/dokan-dev/dokany/releases/tag/v1.2.1.2000 is out with a bumbed DOKAN_VERSION. That should be used instead.

**Comment by taruti** [comment]
i.e. check the return value of ```C.kbfsLibDokan_GetVersion(dokanVersionProc) >= 121```.

**Pull request closed without merging**